VndbSharp/VndbUtils.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using VndbSharp.Structs.Models.Dumps;
+
+namespace VndbSharp
+{
+    public class VndbUtils
+    {
+        protected DateTime LastTime = DateTime.Now;
+        protected static Boolean FirstRun = true;
+
+        public async Task<IEnumerable<Tag>> GetTagDumpAsync()
+            => await this.GetDumpAsync<IEnumerable<Tag>>(Constants.TagsDump, null).ConfigureAwait(false);
+
+        public async Task<IEnumerable<Trait>> GetTraitDumpAsync()
+            => await this.GetDumpAsync<IEnumerable<Trait>>(Constants.TraitsDump, null).ConfigureAwait(false);
+
+        protected async Task<T> GetDumpAsync<T>(String dumpUrl, String userAgent) where T : class
+        {
+            //sets a delay of 1 minute between fetching each dump
+            if (!FirstRun && DateTime.Now - LastTime < TimeSpan.FromSeconds(60))
```

Comment:
Based off of the response of yorhel, this actually isn't needed. If you want i can remove it when i add Client / Version support into the main `VndbClient` class.